tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -61,8 +62,10 @@
 #define BTP_SERVICE_ID_VCP	20
 #define BTP_SERVICE_ID_CAS	21
 #define BTP_SERVICE_ID_MCP	22
+#define BTP_SERVICE_ID_GMCS	23
+#define BTP_SERVICE_ID_MCS	24
```

Comment:
I'd not define BTP_SERVICE_ID_MCS until it is used, so just remove it

Reply:
Fixed